### PR TITLE
Topic/scene limit

### PR DESCRIPTION
**Pull request by danzelenak-usgs**

### CODE REVIEW

**Comment by danzelenak-usgs** [comment]
I'm not sure this will work...opening the PR now to get the ball rolling on code review.

File: api/domain/order.py [hunk 1]
```diff
@@ -244,7 +244,7 @@ def find(cls, id):
         """
         if isinstance(id, int):
             found = cls.where({'id': id})
-        elif isinstance(id, basestring):
```
Comment (caustin-usgs) [inline api/domain/order.py]: what was this change for?

Comment (danzelenak-usgs) [reply api/domain/order.py]: basestring is deprecated in favor of str.  it's not related to the topic, but it showed up in pycharm linting.

File: test/test_production_api.py [hunk 2]
```diff
@@ -78,6 +80,27 @@ def test_fetch_production_products_plot(self):
         self.assertTrue(response is True)
         self.assertEqual(len(pscene), 1)
```
Comment (caustin-usgs) [inline test/test_production_api.py]: I don't think this test belongs in the test_production_api test module.   tests here should be for functionality exercised in the production side of the api. this is on the user side, while placing an order

**Pull request merged**